src/coverup/utils.py

```diff
@@ -85,7 +85,11 @@ async def subprocess_run(args: str, check: bool = False, timeout: T.Optional[int
     except asyncio.TimeoutError:
         process.terminate()
         await process.wait()
-        raise subprocess.TimeoutExpired(args, timeout) from None
+        if timeout:
+            timeout_f = float(timeout)
+        else:
+            timeout_f = 0.0
+        raise subprocess.TimeoutExpired(args, timeout_f) from None
```

Comment:
Can `TimeoutError` be raised if `timeout` is `None`?

Reply:
This was to mollify mypy (which still barks at a lot of things in coverup).

Reply:
Yea, well, mypy is broken (doesn't know any better).